### PR TITLE
Add Kong Ingress guide

### PR DESCRIPTION
**Pull request by FranBarrera**

### CODE REVIEW

**Comment by FranBarrera** [comment]
@sameersbn @sebgoa I'm not sure where this guide should be, I've added to guides but I haven't modified the index yet, so let me know if guides is ok or if I should move it to another place.

**Comment by odacremolbap** [comment]
@FranBarrera might this be missing the link at [mkdocs.yml](https://github.com/triggermesh/docs/blob/c3690ad7fbe4228dc92dd894fa0c3ebe7ce3d132/mkdocs.yml#L44-L57)?

**Comment by odacremolbap** [comment]
Also, I think you are using CR/LF instead of letting the rendered web page to manage that. Or maybe that is some configuration at your IDE. For example:

```
Once Kong is installed,
you should note down the IP address or public CNAME of
`kong-proxy` service.
```

should be

```
Once Kong is installed, you should note down the IP address or public CNAME of `kong-proxy` service.
```

**Comment by FranBarrera** [comment]
> @FranBarrera might this be missing the link at [mkdocs.yml](https://github.com/triggermesh/docs/blob/c3690ad7fbe4228dc92dd894fa0c3ebe7ce3d132/mkdocs.yml#L44-L57)?

Forgot to push the last commit, it should be good now.

File: docs/guides/kong-ingress.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+We will configure Knative to use `kong` as the Ingress class:
+
+```console
+kubectl patch configmap/config-network \
+  --namespace knative-serving \
+    --type merge \
+      --patch '{"data":{"ingress.class":"kong"}}'
+```
+
+## Setup Knative domain
```
Comment (odacremolbap) [inline docs/guides/kong-ingress.md:56]: Sub-item element +
Capital letters at titles

```suggestion
### Setup Knative Domain
```

File: docs/guides/kong-ingress.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+```console
+kubectl -n kong get svc kong-proxy
+
+NAME         TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
+kong-proxy   LoadBalancer   10.98.223.191   35.141.22.45     80:30119/TCP,443:31585/TCP   1m
+```
+
+In the example above the external IP address `35.141.22.45` was provisioned.
+
+## Configure Knative to use Kong for Ingress
```
Comment (odacremolbap) [inline docs/guides/kong-ingress.md:43]:
```suggestion
## Configure Kong Network Layer For Knative
```

**Review by odacremolbap — COMMENTED** [review]
Getting there!

what could be  some improvements commented.

**Review by antoineco — APPROVED** [review]

**Pull request merged**